modules/ocf_puppet/manifests/puppetserver.pp

```diff
@@ -39,6 +39,18 @@
     notify               => Service['puppetserver'],
   }
 
+  # Allow kubernetes secrets to access kubernetes secrets
+  puppet_authorization::rule { 'kubernetes-secrets':
+    match_request_path   => '^/puppet/v3/file_(content|metadata)s?/kubernetes-secrets$',
```

Comment:
Why do we need content and metadata? I imagine one is for `get` and the other for `post`, though we probably don't need to `post` from masters, and mentioned in a previous comment